code/menuui/techmenu.cpp

```diff
@@ -5,7 +5,8 @@
  * or otherwise commercially exploit the source or things you created based on the 
  * source.
  *
-*/ 
+*/ k to the state before "git rebase", run "git rebase --abort".
+
```

Comment:
???

Reply:
I think a few errant lines got in there.  I was in a rush, sorry